src/GraphQL/Mutations/CreateListingMutation.php

```diff
@@ -64,10 +63,6 @@ public function type(): Type
     public function args(): array
     {
         return [
-            'account' => [
```

Comment:
`signingAccount` field by default is optional, should we make it required like `account`

Reply:
Why is an account required? I'm not even sure why `account` did exist maybe I missed something, do you know?

Reply:
Right, `account` doesn't appear to be actually used in the mutation or encodable data etc.  We should remove that and use the optional `signingAccount` flow to keep it consistent with all the other mutations 👍🏻